install

```diff
@@ -86,4 +87,23 @@ install() {
   echo "$INDENT $ swarmlet logs loadbalancer_traefik -f"
 }
 
-install
+minimal-install() {
+  echo "$PREFIX Installing minimal Swarmlet"
+
+  echo "$PREFIX Cloning Swarmlet from $REPOSITORY"
+  git clone -q $REPOSITORY $SWARMLET_INSTALL_ROOT
+
+  echo "$PREFIX Linking executable"
+  ln -s $SWARMLET_INSTALL_ROOT/swarmlet /usr/local/sbin/
+
+  echo "$PREFIX Initializing minimal Swarmlet"
+  swarmlet minimal-init "$@"
+
+  echo "$PREFIX Installation complete"
+}
+
+if [ "$MINIMAL_INSTALL" = true ]; then
```

Comment:
If we're going with `INSTALLATION_MODE=<mode>`, maybe let's turn this into a switch statement.
Something like:
```sh
case $INSTALLATION_MODE in
  full) install "$@" ;;
  minimal) minimal-install "$@" ;;
  othermode) othermode-install "$@" ;;
  *) echo -e "$PREFIX Unknown installation option $INSTALLATION_MODE\nExiting"; exit 1 ;;
esac
```

Reply:
:+1: